recipes/klayout/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set version = "0.27.12" %}
+
+package:
+  name: klayout
+  version: {{ version }}
+
+source:
+  url: https://github.com/KLayout/klayout/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6ef5e6ae5ea387618663949a1efa105406aeb36db2d18d89eff0345955aae979
+  patches:
+    - force-python-dynamic-link.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports:
+    - qt
+  missing_dso_whitelist:  
+    - "*/libnet_tracer.so.0"
+    - "*/liblefdef.so.0"
+    - "*/libpcb.so.0"
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ compiler('c') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxau') }}  # [linux]
+    - {{ cdt('libxcb') }}  # [linux]
+    - {{ cdt('expat') }}  # [linux]
+    - {{ cdt('libpng') }}  # [linux]
+
+  host:
+    - ruby >2
+    - qt >=5.0,<6.0
+  run:
+    - qt-main
+    - python
+    - libzlib
+    - libpython-static
+    - expat
+    - libexpat
+    - curl  # [win]
+    - pthreads-win32  # [win]
+
+test:
+  commands:
+    - klayout -v
+
+about:
+  home: https://www.klayout.de/
+  summary: "Your Mask Layout Friend"
+  description: |
+    - View: Fast and accurate viewing of huge layout files
+    - Edit: Draw, modify and transform hierarchical layout
+    - Generate: Script layout generators, PCells and layout transformation tools
+    - Analyze: Search, verify, and measure layout and code analysis scripts
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file: LICENSE
+  doc_url: https://www.klayout.de/doc.html
+  dev_url: https://github.com/KLayout/klayout
+
+extra:
+  recipe-maintainers:
+    - basnijholt
```

Comment:
@curtisma, @klayoutmatthias, @proppy, @flaport, @joamatab, do any of you want to be added as feedstock maintainers here?

I have been able to successfully compile for MacOS and Linux using inspiration from previous PRs.

At the moment I do not care about Windows. Anyone who wants to add that is welcome to do so once this PR is merged.

Reply:
We do have some packages here as well: https://github.com/hdl/conda-eda/tree/master/misc/klayout (not sure if that was part of the inspiration)

Feel free to add me as a co-maintainer.

Reply:
/cc @xobs

Reply:
@proppy, the recipe in the link you provided helped a lot! I will add you.

Reply:
note we also have https://github.com/conda-forge/magic-feedstock in conda-forge, in case you're interested.

Reply:
> @curtisma, @klayoutmatthias, @proppy, @flaport, @joamatab, do any of you want to be added as feedstock maintainers here?
> 
> I have been able to successfully compile for MacOS and Linux using inspiration from previous PRs.
> 
> At the moment I do not care about Windows. Anyone who wants to add that is welcome to do so once this PR is merged.

Yeah, I'd like to be a feedstock maintainer.

Thanks for the great help here!